### PR TITLE
Linking image of fossasia project to website

### LINKED ISSUE #507: Link Project website URL to images of foassasia projects section
**Actual Behaviour**
FOR EXAMPLE

screenshot_2018-11-25 google code-in with fossasia 2018 6

**Expected Behaviour**
 
clicking on image should open the project website page 
for example
for melix .->https://meilix-generator.herokuapp.com/

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by subhahu123**
# Changes done in this Pull Request
changing the orgprojects.yml file of data folder to add url to images of fossasia projects

- Fixes #507

### CODE REVIEW

**Comment by subhahu123** [comment]
@nikit19  i m facing issue with my github url it's saying page build is failed and i made on it are not applied when i try to open a website
that's why i currently unable to show a preview link

File: index.html [hunk 1]
```diff
@@ -410,7 +410,9 @@ <h3>FOSSASIA | Projects</h3><br>
     {% for projects in site.data.orgprojects %}
     <div class="row v-align-children extra_padding_b">
       <div class="col-md-7 col-sm-6 text-center mb-xs-24">
+        <a href="{{projects.url}}">
         <img class="cast-shadow" alt="Screenshot" src="img/{{projects.img}}" />
```
Comment (Ryan10145) [inline index.html]: Indent the line `Screenshot`

Comment (subhahu123) [reply index.html]: done

**Comment by subhahu123** [comment]
@abishekvashok  so u want that clicking on image and try should open same website

**Pull request closed without merging**